bootstrap/kubeadm/main.go

```diff
@@ -123,6 +125,12 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&syncPeriod, "sync-period", 10*time.Minute,
 		"The minimum interval at which watched resources are reconciled (e.g. 15m)")
 
+	fs.Float32Var(&rateLimitQPS, "rate-limit-qps", 20,
+		"The number of queries per second that should be allowed")
```

Comment:
Should make it clear here, and elsewhere, that this impacts the queries from this controller to the apiServer.

Reply:
Should also ensure this is aligned with the work at https://github.com/kubernetes-sigs/cluster-api/pull/8411 which is adding those flags for clusterctl.

Reply:
Maybe `restConfigQPS` and `restconfigBurst` are the clearest names, with the description pointing them out as being configurations for the Kubernetes client?

Reply:
Makes perfect sense, thanks! I will update.
I was not aware of #8411, will take a look :slightly_smiling_face: